packages/react-ui/src/app/features/flows/flows-columns.tsx

```diff
@@ -119,7 +121,7 @@ export const createColumns = (
       return (
         <div className="w-fit px-3 py-1 flex items-center gap-1 rounded-full border border-warning bg-warning-100">
           <CircleAlert size={16} className="text-warning" />
-          <span className="font-medium text-sm text-blueAccent-200 dark:text-primary text-nowrap">
+          <span className="font-regular text-sm text-blueAccent-200 dark:text-primary text-nowrap">
```

Comment:
there is no `font-regular` tailwind
<span className="font-regular text-sm text-blueAccent-200 dark:text-primary text-nowrap">
```suggestion
          <span className="font-normal text-sm text-blueAccent-200 dark:text-primary text-nowrap">
```